runhouse/servers/obj_store.py

```diff
@@ -572,11 +580,40 @@ def delete_for_env_servlet_name(env_servlet_name: str, key: Any):
     def delete_local(self, key: Any):
         self.pop_local(key)
 
+    def delete_env(self, key: Any):
+        from runhouse.globals import env_servlets
+
+        # clear keys in the env servlet
+        env_servlet_keys = list(self._kv_store.keys()) if self.has_local_storage else []
```

Comment:
these are just the keys within this servlet, unrelated to the servlet that we are trying to delete, right? why would we mark these keys as deleted? bit confused